src/main/java/eu/europeana/metis/sandbox/config/SandboxConfig.java

```diff
@@ -67,6 +69,21 @@ private String edmSorterUrl() throws IOException {
     return edmSorterUrl;
   }
 
+  @Bean
+  @Scope(ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+  XsltTransformer xsltTransformer(String datasetName, String edmCountry, String edmLanguage)
+      throws IOException, TransformationException {
+    return new XsltTransformer(defaultXsltTransformerUrl(), datasetName, edmCountry, edmLanguage);
+  }
+
+  private String defaultXsltTransformerUrl() throws IOException {
+    if (defaultXsltUrl == null) {
+      defaultXsltUrl = resourceLoader.getResource("classpath:edm/default.xslt.xsl").getURL()
```

Comment:
You are including the default xslt as a file in this repository. Wouldn't it be preferable to be more flexible? Just like we can upload a new xslt in Metis at any time, we should allow the xslt to be changed at runtime here as well. Perhaps have a line in the config file with a file location? Or perhaps some other solution? @stzanakis , you have any ideas?

Reply:
Because this is an "open" service and we do not have authentication, I would opt out of having an upload of a default xslt.
That being said though, I agree we should have it configurable and change it(if required).
To do so it can either be a configuration property pointing to a path on the file system but that implies that the default xslt will be packaged with the .war file(more likely from jenkins) and a new xslt will require a new build. That is because of bluemix deployments limitations.
Or it has to be part of the database and loaded from there on a restart or a trigger/reload call.

p.s. generally the default xslt changes rarely so we can opt in on any of the options.

Reply:
is there a centralized place where you pull it ? a Url that I can use? Or how is it managed today ?
I could remove it from the repository and it can be injected by the deployment like the properties are, so that is not in the repo and it can change separately

Reply:
In metis after the first deployment of metis-core there is an endpoint which we use to upload the xslt and it is inserted in the db. But there we have authentication/authorization in place so it's fine to do so.
The metis-core also supplies another endpoint to GET the latest default xslt without using authentication, you could use that but that required the service to be up and running.
Perhaps locally handling it is better..

Reply:
In that case I think I like the option of providing the file during build/deployment like the application.yml, if that is possible, since the story says 
> EDM records are transformed against the default XSLT transformation used by Metis. The user has no option for uploading his own XSLT
And because the default does not change that often I think the ability to switch by deploying will be good enough for now. 
Tradeoff of handling locally is that we need to remember that the sandbox has to be deployed in order to apply changes.
To summarize, proposal is to remove the xstl from the repo, and let it be provided by jenkins build